src/components/titlePanel/createTitlePanel.ts

```diff
@@ -0,0 +1,90 @@
+import { VNodeProperties } from '@dojo/interfaces/vdom';
+import createWidgetBase from '@dojo/widget-core/createWidgetBase';
+import { v } from '@dojo/widget-core/d';
+import { DNode, Widget, WidgetFactory, WidgetProperties } from '@dojo/widget-core/interfaces';
+import themeable, { ThemeableMixin } from '@dojo/widget-core/mixins/themeable';
+
+import * as css from './styles/titlePanel.css';
+import * as animations from '../../styles/animations.css';
+
+export interface TitlePanelProperties extends WidgetProperties {
+	collapsed?: boolean;
```

Comment:
I'd recommend swapping `collapsed` for `open` as we've used `open` elsewhere and it seems that you're needing to negate `collapsed` for most use cases.